grid2op/tests/test_ma.py

```diff
@@ -22,15 +23,22 @@
 class MATesterGlobalObs(unittest.TestCase):
     def setUp(self) -> None:
         
-        with warnings.catch_warnings():
-            warnings.filterwarnings("ignore")
-            self.env = make("educ_case14_storage", test=True,
-                            action_class=PlayableAction, _add_to_name="test_ma")
-
         self.action_domains = {
             'agent_0' : [0,1,2,3, 4],
             'agent_1' : [5,6,7,8,9,10,11,12,13]
         }
+        
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore")
+            
+            custom_params = Parameters()
```

Comment:
nope... 

custom_params = self.cent_env.parameters

Why do you start from scratch ?